src/sentry/post_process_forwarder/post_process_forwarder.py

```diff
@@ -31,6 +34,7 @@ def __init__(
         max_batch_size: int,
         max_batch_time: int,
         concurrency: int,
+        eventstream_type: str | None = None,
```

Comment:
I think this needs to be passed in actually as static args for the post process forwarders: 

shoud be here https://github.com/getsentry/sentry/blob/master/src/sentry/consumers/__init__.py#L370-L389

and they can be passed in as static args.